website/docs/r/appautoscaling_target.html.markdown

```diff
@@ -44,7 +44,7 @@ resource "aws_appautoscaling_target" "dynamodb_index_read_target" {
 resource "aws_appautoscaling_target" "ecs_target" {
   max_capacity       = 4
   min_capacity       = 1
-  resource_id        = "service/clusterName/serviceName"
+  resource_id        = "service/${var.cluster_name}/${var.service_name}"
```

Comment:
We should probably provide the resource references here as well: `service/${aws_ecs_cluster.example.name}/${aws_ecs_service.example.name}`